ntbs-service/Services/AuthorizationService.cs

```diff
@@ -217,8 +217,7 @@ public async Task<IList<AlertWithTbServiceForDisplay>> FilterAlertsForUserAsync(
             alerts = alerts.Where(a => userTbServiceCodes.Contains(a.TbServiceCode)).ToList();
             if (userType == UserType.PheUser)
             {
-                alerts = alerts.Where(a => a.AlertType != AlertType.TransferRequest &&
-                                           a.AlertType != AlertType.TransferRejected).ToList();
+                alerts = alerts.Where(a => a.AlertType != AlertType.TransferRejected).ToList();

```

Comment:
I know this isn't directly covered by this ticket, but it does seem strange to exclude transfer rejections from regional users. Any idea why we were doing this/whether we still want to do it now?

Reply:
From memory, I seem to remember the filtering was to: 1. not have a huge list of alerts on the homepage for regional users (which we may be tackling with Power BI now anyway) and 2. probably likely the a TB service user requested the transfer so having a regional user dismiss the alert would mean they might not see it

Reply:
Ok cool - it would be good to add a comment here explaining that we're doing this because of point 2 (I don't think point 1 is so relevant because regional users do see a huge list of alerts, and I don't expect rejected transfers to be that common anyway)

Reply:
Removed filtering